docker/builder.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -f setup.py ]; then
```

Comment:
nit: Since you're already guaranteed bash, I'd recommend using `[[`.

Reply:
Does it improve things much?

Reply:
@BYK `[` will not be a shell builtin in some cases (on my mac, bash resolves it to `/usr/local/opt/coreutils/libexec/gnubin/[`), so yes, it is considerably faster since you avoid subprocessing. In general if there is a bash shebang, I try and use as much bash builtins as I can.

Reply:
TIL. Thanks a lot @joshuarli  🙂